cellxgene_schema_cli/cellxgene_schema/validate.py

```diff
@@ -1315,7 +1335,92 @@ def _check_column_availability(self):
 
     def _check_spatial(self):
         """
-        Validate spatial-related values of the AnnData object. Validation is not defined in schema definition yaml.
+        Sequence validation of spatial-related values of the AnnData object.
+
+        :rtype none
+        """
+        self._check_spatial_uns()
+        self._check_spatial_obs()
+
+    def _check_spatial_obs(self):
+        """
+        Validate obs spatial-related values of the AnnData object. Validation is not defined in schema definition yaml.
+        Errors are added to self.errors.
+
+        :rtype none
+        """
+
+        # Exit if obs is not specified. Error is reported in core validate functionality.
+        obs_component = getattr_anndata(self.adata, "obs")
+        if obs_component is None:
+            return
+
+        # obs spatial validation is dependent on uns.spatial.is_single; exit if not specified. Errors are
+        # reported downstream.
+        uns_component = getattr_anndata(self.adata, "uns")
+        if uns_component is None or "spatial" not in uns_component or "is_single" not in uns_component["spatial"]:
+            return
```

Comment:
per brian mott's comment, I think we need to remove this block + run an additional check in _validate_spatial_tissue_position to enforce that the input tissue_position col is not present if spatial is not in uns_component